### DISCUSSION TITLE
更新site.json后没生效

### DISCUSSION THREAD

**Discussion by seekexplore** [spencerwooo/onedrive-vercel-index#265] [Q&A]
如题，在仓库修改配置文件后在vercel项目点击重新部署，网站没有发生改变
而且从vercel的来源去到github仓库
![image](https://user-images.githubusercontent.com/97540035/149080397-75ac263f-750f-4e46-bb9f-3edf010ceb8f.png)
发现仓库到达的分支不是main而是莫名其妙出现了另外一个
![image](https://user-images.githubusercontent.com/97540035/149080617-be0a995e-3b11-4633-af04-b51741a43cef.png)
另外的分支保持着一开始部署时的配置状态，对这个分支进行修改显示
![image](https://user-images.githubusercontent.com/97540035/149080862-b9c0ee5f-4dd8-4c86-8c56-bab5b773fec9.png)
请问是什么原因导致？我没有选择直接fork项目而是Import your project to GitHub

**Comment by spencerwooo** [comment]
得保证 Vercel 是从 GitHub 项目的 main 分支部署的，建议重新在 Vercel 里面 import 一下你的 repo

**Reply by seekexplore** [reply]
你好，如图所示我在vercel显示的分支确实是main
![image](https://user-images.githubusercontent.com/97540035/149088050-8d417979-86b8-4fe1-83a9-f8861b07adac.png)
但点击Update site.json跳转到github显示的是第一次修改site.json的修改内容而不是最新修改内容

**Reply by spencerwooo** [reply]
你在 GitHub 这边修改 site.json 啊，你不会是在 Vercel 那边修改的吧？

**Reply by spencerwooo** [reply]
直接在 GitHub 这边的 main 分支改，改完之后 Vercel 会自动部署修改后的 GitHub 仓库代码

**Reply by seekexplore** [reply]
我是在github修改的，而且vercel机器人也在GitHub最新修改的记录里面回复了，但是还是没有修改成功
![image](https://user-images.githubusercontent.com/97540035/149134529-e5875a47-8bd8-47cf-838d-0daedf18eb58.png)
